media/js/plugincheck/tmpl/plugincheck-redesign.ui.tmpl.js

```diff
@@ -0,0 +1,45 @@
+/* exported featuredPluginsTmpl, otherPluginsTmpl, unknownPluginsTmpl */
+
```

Comment:
These are exceptions for ESLint parsing, which need to be generated manually. Really we shouldn't be passing round global variables like this, which is why it complains. I'm not going to the effort of fixing this here since it's only related to this one page (plugincheck is also to become deprecated in the future anyway).